pkg/generate/app/cmd/newapp.go

```diff
@@ -29,6 +29,13 @@ import (
 	"github.com/openshift/origin/pkg/util/namer"
 )
 
+const GeneratedByNamespace = "openshift.io/generated-by"
+
+var (
+	GeneratedByNewApp   = map[string]string{GeneratedByNamespace: "OpenShiftNewApp"}
```

Comment:
I can, I wanted to have this on one place so you know how to change it :-) But you're right.